apis/v1alpha1/cluster_tc_program_types.go

```diff
@@ -37,7 +37,7 @@ type ClTcAttachInfo struct {
 	// program. If Containers is not specified, the BPF program will be attached
 	// in the root network namespace.
 	// +optional
-	Containers *ClContainerSelector `json:"containers"`
+	Containers *ClContainerSelector `json:"containers,omitempty"`
```

Comment:
This will become NetworkNamespaces with my netns pr.  Also, I don't think it's valid to have NetworkNamespaces set when we also have interface discovery enabled, because interface discovery should also be reporting the network namespace in which the interface was discovered.  However, I suggest we get this pr and my netns pr merged and then we can figure out how best to do this validation.